src/main/java/fi/aalto/cs/apluscourses/intellij/model/SettingsImporter.java

```diff
@@ -66,8 +66,8 @@ public void importIdeSettings(@NotNull Course course) throws IOException {
     String configPath = FileUtilRt.toSystemIndependentName(PathManager.getConfigPath());
     StartupActionScriptManager.addActionCommands(
         Arrays.asList(
-            new StartupActionScriptManager.UnzipCommand(file, new File(configPath)),
-            new StartupActionScriptManager.DeleteCommand(file)
+            new StartupActionScriptManager.UnzipCommand(file.toPath(), Path.of(configPath)),
```

Comment:
take a look @OlliKiljunen

Reply:
You changed `File` to `Path` (makes sense) but also reversed the order of commands? Why?

Reply:
Sorry, I looked wrong. nevermind. all good.